cookbooks/fb_syslog/recipes/default.rb

```diff
@@ -73,6 +73,12 @@
   notifies :restart, "service[#{service_name}]"
 end
 
+directory '/etc/rsyslog.d' do
+  action :delete
+  recursive true
+  notifies :restart, "service[#{service_name}]"
+end
+
```

Comment:
Can you spin this out in a separate PR ? I just checked and we still have a bunch of crap in there so it'd need a careful rollout.

Reply:
Alternatively, gate it behind an attribute knob